Gopkg.toml

```diff
@@ -40,6 +40,11 @@ required = [
     name = "k8s.io/gengo"
     unused-packages = false
 
+  # Don't prune code needed by openapi-gen
+  [[prune.project]]
+    name = "k8s.io/kube-openapi"
+    unused-packages = false
```

Comment:
Shouldn't the required binaries be added to the required section above like client-gen, conversion-gen, and apiregister-gen instead of overriding pruning?

Reply:
When I add `"k8s.io/kube-openapi"` to the required section and run `dep ensure -v` it fails:

```
Solving failure: No versions of k8s.io/kube-openapi met constraints:
	master: Could not introduce k8s.io/kube-openapi@master, as its subpackage k8s.io/kube-openapi does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	master: Could not introduce k8s.io/kube-openapi@master, as its subpackage k8s.io/kube-openapi does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	feature-serverside-apply: Could not introduce k8s.io/kube-openapi@feature-serverside-apply, as its subpackage k8s.io/kube-openapi does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	release-1.10: Could not introduce k8s.io/kube-openapi@release-1.10, as its subpackage k8s.io/kube-openapi does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
	release-1.8: Could not introduce k8s.io/kube-openapi@release-1.8 due to multiple problematic subpackages:
	Subpackage k8s.io/kube-openapi does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)	Subpackage k8s.io/kube-openapi/pkg/util/proto is missing. (Package is required by k8s.io/apimachinery@release-1.9.)
	release-1.9: Could not introduce k8s.io/kube-openapi@release-1.9, as its subpackage k8s.io/kube-openapi does not contain usable Go code (*build.NoGoError).. (Package is required by (root).)
```